### ISSUE TITLE
Can't use Nimble on Windows 10

### ISSUE THREAD

**Issue by ipelekhan** [nim-lang/nimble#762]
Hi, sorry if it's not good place to report. Installed Nim recently using instruction from the official website. Compiler itself works fine, was able to start doing some tutorials and compiling tools, `nimpretty` for example. Plugins for VS Code work as well without any issues. The only thing is not working is package manager. Tried to run it through `cmd` and Git Bash, no output, no errors, nothing. Check the screenshot:

![image](https://user-images.githubusercontent.com/2616427/72066435-df619980-32e0-11ea-95bd-685f9595c3b7.png)

I also disabled antivirus/Windows Defender/checked firewalls just in case. And tried to run the same things as administrator. Tried to build `nimble` with `koch`, which was done successfully. Issue persists.

Any ideas how to debug this?

**Comment by Araq** [comment]
Is this a cygwin related problem? Can you show me a screenshot for ``cmd.exe``?

**Comment by dom96** [comment]
Very peculiar. You might also wish to give choosenim a shot.

**Comment by inv2004** [comment]
@ipelekhan 
I have fresh nim on windows-10 and everything looks fine.
Is it possible that something wrong with nimble output? or maybe you have alias which shadow output or something like that.

Regards,

**Comment by ipelekhan** [comment]
Here's screenshot from `cmd`:
![image](https://user-images.githubusercontent.com/2616427/72245831-b6e3e300-35f1-11ea-8bd6-5c4b8b741ec3.png)

Tried to run `choosenim` self extracting installer and it surprisingly behaved the same way.
That made me to try to run Windows 10 in Virtual Box. Nim tools worked fine there. 
So I have decided to close this issue, as it seems like there is another security software prevents running executables which I can't find/disable (it's a company's laptop, so it's possible that there are some spyware installed apart from defender and antivirus). Thanks for your time.
